src/mod2c_core/nocpout.c

```diff
@@ -1908,7 +1908,7 @@ static void defs_h(s)
 	Item *q;
 	
 	if (s->subtype & ARRAY) {
-		Sprintf(buf, "#define %s (_p + %d*_STRIDE)\n", s->name, parraycount);
+		Sprintf(buf, "#define %s (_p + %d*_STRIDE*%d)\n", s->name, parraycount, s->araydim);
```

Comment:
araydim ? or arraydim?

Reply:
don't know why but that is declared as `araydim` : https://github.com/BlueBrain/mod2c/blob/469c74dc7d96bbc5a06a42696422154b4cd2ce28/src/mod2c_core/modl.h#L147